src/lib/components/workflow-raw-history-link.svelte

```diff
@@ -3,23 +3,25 @@
   import { translate } from '$lib/i18n/translate';
   import { isCloud } from '$lib/stores/advanced-visibility';
   import { parameters } from '$lib/stores/events';
-  import { routeForWorkflowHistoryJson } from '$lib/utilities/route-for';
+  import { routeForEventHistory } from '$lib/utilities/route-for';
 
   const { namespace, workflowId, runId } = $parameters;
 
-  $: href = routeForWorkflowHistoryJson({
+  $: href = routeForEventHistory({
```

Comment:
We can trash routeForWorkflowHistoryJson if it's not used anymore

Reply:
done 👍